bindata/assets/kube-scheduler/pod.yaml

```diff
@@ -120,6 +120,9 @@ spec:
   priorityClassName: system-node-critical
   tolerations:
   - operator: "Exists"
+  - key: "node-role.kubernetes.io/control-plane"
+    operator: "Exists"
+    effect: "NoExecute"
```

Comment:
`- operator: "Exists"` tolerates any taint.

Reply:
Thanks! Will take note of that